DNN Platform/Website/Resources/Shared/components/CountriesRegions/dnn.CountriesRegions.js

```diff
@@ -116,7 +116,13 @@ function setupCountryAutoComplete() {
 			} else {
 				clearCountryValue(this);
 			}
-		}
+        },
+        focus: function (event, ui) {
+            event.preventDefault();
+            this.value = ui.item.name;
+            $(this).attr('data-value', ui.item.id);
+            $(this).attr('data-text', ui.item.name);
+        }
```

Comment:
Adjusting spaces to tabs (to match the rest of the file)
```suggestion
		},
		focus: function (event, ui) {
			event.preventDefault();
			this.value = ui.item.name;
			$(this).attr('data-value', ui.item.id);
			$(this).attr('data-text', ui.item.name);
		}
```